bin/rhsm-service

```diff
@@ -34,6 +34,7 @@ if __name__ == "__main__":
             objects.RegisterDBusObject,
             objects.AttachDBusObject,
             objects.ProductsDBusObject,
+            objects.UnRegisterDBusObject,
```

Comment:
Please change the class name to `UnregisterDBusObject`.  `UnRegister` looks odd.  I realize it's `UnRegister` in `managercli` but that's weird too and if you wouldn't mind fixing it there also, I'd appreciate it.